resources/views/settings/tags.blade.php

```diff
@@ -94,3 +102,15 @@
 </div>
 
 @endsection
+<script>
+  function toggleEditInput(editIcon) {
+    let tableRow = editIcon.closest('.table-row');
+    var tagName = tableRow.querySelector('.tag-name');
```

Comment:
Would mind you replacing all the other `var`s with `let`s as well? Thanks.

Reply:
done